packages/components/src/components/popover/popover.tsx

```diff
@@ -107,4 +128,20 @@ export class Popover {
       this.button.focus();
     }
   };
+
+  private detectNativeCase = (): void => {
+    const supportsNativePopover = HTMLElement.prototype.hasOwnProperty('popover');
```

Comment:
Can't we detect this once in utils since this won't change on a per component basis?